lib/data_freshness/requirements.txt

```diff
@@ -0,0 +1 @@
+pngme-api >= 0.5.0
```

Comment:
You can restrict the minimum version here to be `0.9.2` since it requires the version which supports passing the `page` param
```suggestion
pngme-api >= 0.9.2
```

Reply:
Nice!